akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -44,18 +45,18 @@ final case class GraphStageModule[+S <: Shape @uncheckedVariance, +M](
 /**
  * INTERNAL API
  */
-object GraphStages {
+@InternalApi private[akka] object GraphStages {
```

Comment:
Yes. You are right. This is an artifact of the find and replace. But it does not hurt to have it below as well. Then you do not need context to see that these are internal apis when browsing the source code.